"Unable to start service com.flowcrypt.email.service.PassPhrasesInRAMServ" errors

Seems like similar issues was fixed earlier (https://github.com/FlowCrypt/flowcrypt-android/issues/1588), but there are still many error reports:
```
java.lang.RuntimeException: Unable to start service com.flowcrypt.email.service.PassPhrasesInRAMService@6e4302 with null: android.app.ForegroundServiceStartNotAllowedException: Service.startForeground() not allowed due to mAllowStartForeground false: service com.flowcrypt.email/.service.PassPhrasesInRAMService
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5286)
at android.app.ActivityThread.-$$Nest$mhandleServiceArgs(Unknown Source:0)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2531)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:230)
at android.os.Looper.loop(Looper.java:319)
at android.app.ActivityThread.main(ActivityThread.java:8893)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
Caused by: android.app.ForegroundServiceStartNotAllowedException: Service.startForeground() not allowed due to mAllowStartForeground false: service com.flowcrypt.email/.service.PassPhrasesInRAMService
at android.app.ForegroundServiceStartNotAllowedException$1.createFromParcel(ForegroundServiceStartNotAllowedException.java:54)
at android.app.ForegroundServiceStartNotAllowedException$1.createFromParcel(ForegroundServiceStartNotAllowedException.java:50)
at android.os.Parcel.readParcelableInternal(Parcel.java:4882)
at android.os.Parcel.readParcelable(Parcel.java:4864)
at android.os.Parcel.createExceptionOrNull(Parcel.java:3064)
at android.os.Parcel.createException(Parcel.java:3053)
at android.os.Parcel.readException(Parcel.java:3036)
at android.os.Parcel.readException(Parcel.java:2978)
at android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:7214)
at android.app.Service.startForeground(Service.java:775)
at com.flowcrypt.email.service.PassPhrasesInRAMService.onStartCommand(PassPhrasesInRAMService.kt:60)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5268)
```

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/PrivateKeyDetailsFragmentPassInRamFlexibleTimeFlowTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/PrivateKeyDetailsFragmentPassInRamFlexibleTimeFlowTest.kt
@@ -96,6 +96,9 @@ class PrivateKeyDetailsFragmentPassInRamFlexibleTimeFlowTest : BaseTest() {
 
   @Test
   fun testInMemoryPassPhraseSessionLengthParameter() {
+    //as tests run a bit differently need to run PassPhrasesInRAMService manually at this stage
+    PassPhrasesInRAMService.start(getTargetContext())
+
     onView(withId(R.id.tVPassPhraseVerification))
       .check(matches(withText(getResString(R.string.pass_phrase_not_provided))))
       .check(matches(hasTextColor(R.color.red)))

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt
@@ -117,7 +117,10 @@ class FlowCryptApplication : Application(), Configuration.Provider {
       val hasTemporaryPassPhrases =
         keysStorage.getRawKeys().any { it.passphraseType == KeyEntity.PassphraseType.RAM }
       if (hasTemporaryPassPhrases) {
-        PassPhrasesInRAMService.start(this)
+        if (GeneralUtil.isAppForegrounded()) {
+          //we can run a foreground service only if the app is visible for a user
+          PassPhrasesInRAMService.start(this)
+        }
       } else {
         PassPhrasesInRAMService.stop(this)
       }

